docs/development.md

```diff
@@ -55,10 +54,40 @@ The box can now be accessed via ssh and the Rails server started directly (this
 ```sh
 vagrant ssh centos7-devel
 cd foreman
-bundle exec foreman start
```

Comment:
Updated to keep this in, I originally removed it in lieu of giving the different options for starting the dev server. Maybe this should be added the quickstart section in the README instead?

Reply:
Given the above lines it's pretty long because it requires some setup. If we can simply out-of-the-box run this without additional config, it's probably worth it but maybe a follow up?